App/Components/ProgressiveImage.tsx

```diff
@@ -34,14 +34,14 @@ export default class ProgressiveImage extends React.Component<ProgressiveImagePr
       <TextileImage
         imageId={this.props.imageId}
         path={'thumb'}
```

Comment:
ah shoot, both of these 'thumbs' should just be `this.props.previewPath` if you want to pop that in

Reply:
Fixed